pkg/quarterdeck/auth.go

```diff
@@ -44,12 +45,13 @@ func (s *Server) Register(c *gin.Context) {
 	ctx := c.Request.Context()
 
 	if err = c.BindJSON(&in); err != nil {
-		log.Warn().Err(err).Msg("could not parse register request")
+		c.Error(err)
```

Comment:
Yeh this was @daniellemaxwell's suggestion as well. And I agree. I'll start making changes so that we start doing:

```
c.Error(fmt.Errorf("could not parse register request: %w", err)) 
``

Which would add the context. 

Sentry has two features: [tags](https://docs.sentry.io/platforms/go/guides/gin/enriching-events/tags/) and [levels](https://docs.sentry.io/platforms/go/guides/gin/usage/set-level/). With these it _might_ be possible to create a mirror that logs to _both_ sentry and zerolog; is that too complicated?